docs/architecture/porting-existing-aspnet-apps/authentication-differences.md

```diff
@@ -7,8 +7,42 @@ ms.date: 11/13/2020
 
 # Comparing authentication and authorization between ASP.NET MVC and ASP.NET Core
 
+In ASP.NET MVC 5, authentication is configured in `Startup.Auth.cs` in the `App_Start` folder. In ASP.NET Core MVC, this configuration occurs in `Startup.cs`. Authentication and authorization are performed using middleware added to the request pipeline in `ConfigureServices`:
+
+```csharp
+// inside Startup.ConfigureServices
+
+app.UseRouting();
+
+app.UseAuthentication();
+app.UseAuthorization();
+
+app.UseEndpoints(endpoints =>
+{
+    endpoints.MapControllerRoute(
+        name: "default",
+        pattern: "{controller=Home}/{action=Index}/{id?}");
+    endpoints.MapRazorPages();
+});
+```
+
+It's important to add the auth middleware in the appropriate location in the middleware pipeline. Only requests that make it to the middleware will be impacted by it. If for instance a call to `UseStaticFiles()` were placed above the code shown here, it would not be protected by authentication and authorization.
+
+In ASP.NET MVC and Web API, applications often would refer to the current user using the `ClaimsPrincipal.Current` property. This property is not set in ASP.NET Core, and any behavior in your app that depends on it will need to [migrate from ClaimsPrincipal.Current](https://docs.microsoft.com/aspnet/core/migration/claimsprincipal-current) by using the `User` property on `ControllerBase` or getting access to the current `HttpContext` and referencing its `User` property. If neither of these is an option, services can request the User as an argument, in which case it must be supplied from elsewhere in the app, or the `IHttpContextAccessor` can be requested and used to get the HttpContext.
```

Comment:
I think this paragraph could be benefit from a list to show the options. Otherwise, the sentences are really long.

```suggestion
In ASP.NET MVC and Web API, applications often refer to the current user using the `ClaimsPrincipal.Current` property. This property isn't set in ASP.NET Core, and any behavior in your app that depends on it will need to [migrate from ClaimsPrincipal.Current](https://docs.microsoft.com/aspnet/core/migration/claimsprincipal-current) by using the `User` property on `ControllerBase` or getting access to the current `HttpContext` and referencing its `User` property. If neither of these solutions is an option, services can request the User as an argument, in which case it must be supplied from elsewhere in the app, or the `IHttpContextAccessor` can be requested and used to get the HttpContext.
```